buildenv/jenkins/common/pipeline-functions

```diff
@@ -78,15 +78,44 @@ def get_sha(REPO, BRANCH) {
 
 def build(BUILD_JOB_NAME, OPENJDK_REPO, OPENJDK_BRANCH, OPENJDK_SHA, OPENJ9_REPO, OPENJ9_BRANCH, OPENJ9_SHA, OMR_REPO, OMR_BRANCH, OMR_SHA, VARIABLE_FILE) {
```

Comment:
Where is `build` called from?  Only by pipelines?  I'm trying to trace who may end up calling `build_with_slack` but the `build` is hard to grep :)

Reply:
yes, only pipelines (i.e nightly builds and eventually OMR-Acceptance)
https://github.com/eclipse/openj9/blob/master/buildenv/jenkins/common/pipeline-functions#L98
https://github.com/eclipse/openj9/blob/master/buildenv/jenkins/common/pipeline-functions#L115